### DISCUSSION TITLE
Can I use ventoy to install Chrome OS?

### DISCUSSION THREAD

**Discussion by ghost** [ventoy/Ventoy#1312] [Q&A] [answered]
Can I use ventoy to install Chrome OS?
(Rammus)

**Comment by ventoy** [comment, marked as answer]
You can use Ventoy to boot CloudReady and FydeOS. https://www.ventoy.net/en/doc_chrome_os.html

What do you mean about `install` ?

**Reply by ghost** [reply]
Ok thanks

**Comment by ejcupcake** [comment]
I had the same issue! I found out you can do that by following this tutorial: https://youtu.be/ROBpXNonVxc
But instead of drag-and-dropping files onto the USB, Use an app like [AnyBurn](anyburn.com) to Edit The Linux mint iso.

**Comment by PranshuZ** [comment]
dont use ventoy for your fydeos or chrome os because you simply can't install it. When i tried it took me directly to login page i.e. the try it option.
to solve it just directly use a new usb and flash it.
